altiProfil/install/www/altiprofil/js/altiProfil.js

```diff
@@ -211,6 +211,13 @@ function getProfil(p1,p2){
 }
 
 function initAltiProfil() {
+    // add altiprofilCtrl = false to all controls with same class as altiProfil
```

Comment:
Do you really need to add a parameter to other controls? You might just need https://github.com/arno974/lizmap-altiProfil/pull/36/files#diff-fc38a3acc81a1c70eb55a3b6657bde18dff636e0fba5a872db37057319d0e841R244 and make `if` tests accordingly.